pointcloud/merging_test.go

```diff
@@ -40,6 +40,7 @@ func makeThreeCloudsWithOffsets(t *testing.T) []CloudAndOffsetFunc {
 }
 
 func TestApplyOffset(t *testing.T) {
+	t.Skip("remove skip once RSDK-1200 improvement is complete")
```

Comment:
```suggestion
	// TODO(RSDK-1200): remove skip when complete
	t.Skip("remove skip once RSDK-1200 improvement is complete")
```
The specific TODO syntax is something we can search/grep on to look for forgotten items, so good to have it.